moc_openshift.py

```diff
@@ -447,6 +447,45 @@ def update_rolebindings(self, project_name, role, rolebindings_json):
         self.logger.debug("payload -> 2: " + json.dumps(payload))
         return self.put_request(url, payload, True)
 
+    # member functions for quotas
+    #Check if Quota exists
```

Comment:
the name of the method is self explanatory, this comment is not adding anything to this code.